utils/kata-manager.sh

```diff
@@ -91,13 +91,19 @@ github_get_latest_release()
 {
 	local url="${1:-}"
 
+	#parses the json response from github. Python is on every machine out there, not jq.
+	local python_jq="import json, sys
+response=json.loads(sys.stdin.read())
+for tag in response:
+    if not ('-' in tag['tag_name']):
+        print(tag['tag_name'])"
 	# Notes:
 	#
 	# - The sort(1) call; none of the standard utilities support semver
 	#   so attempt to perform a semver sort manually.
 	# - Pre-releases are excluded via the select() call.
 	local latest=$(curl -sL "$url" |\
-		jq -r '.[].tag_name | select(contains("-") | not)' |\
+		python3 -c "$python_jq" |\
```

Comment:
I know that most distros (are moving|have moved) to python3, but there might be some that only install python2 by default, which might break this?